README.md

```diff
@@ -87,6 +87,79 @@ module.exports = {
 > `match` option. If you want to automatically create users in Apostrophe,
 > see [creating users on demand](#creating-users-on-demand) below.
 
+### Customizing call to the strategy verify method
+
+At the end of the login process, `@apostrophecms/passport-bridge` will call a method `findOrCreateUser(req, accessToken, refreshToken, profile, callback)`.
```

Comment:
I think it would be more clear to say this instead:

"All passport strategies expect us to provide a `verify` callback. By default, passport-bridge passes its `findOrCreateUser` function as the `verify` callback, which works for many strategies including [strategies here]. For other strategies, you can pass an explicit `verify` option which..."